### PR TITLE
adding relays driver firmware

### PR DESCRIPTION
**Pull request by Basazababo**
Relays driver firmware.
It receives data from RMC and send back relays status to RMC without emonhub configuration

### CODE REVIEW

File: Relays_driver_firmware/Relays_driver_firmware.ino [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+const long check_relays_interval = 2000; 
+
+
+const int relay_pins [] = {9,8,7,6};   // pin number of relays; 
+const byte number_of_relays=4;
+word relays_status [] = {number_of_relays};
+
+void setup()
+{
+  Serial.begin(38400);
```
Comment (jlark) [inline Relays_driver_firmware/Relays_driver_firmware.ino]: Can you make this baud rate a global constant so it' easier to configure. 
Put it at the top of the file in all caps

File: Relays_driver_firmware/Relays_driver_firmware.ino [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+static byte stack[RF12_MAXDATA+4], top, sendLen, dest;           // RF variables 
+static char cmd;
+static word value;                                               // Used to store serial input
+
+unsigned long last_relay_driver_check = 0;        
+const long check_relays_interval = 2000; 
+
+
+const int relay_pins [] = {9,8,7,6};   // pin number of relays; 
+const byte number_of_relays=4;
```
Comment (jlark) [inline Relays_driver_firmware/Relays_driver_firmware.ino:33]: if it's a constant it's  best practice to use all upper case

File: emonFirmware_test_for_relay_driver/emonPi_Serial_LCD.ino [hunk 3]
```diff
@@ -0,0 +1,282 @@
+// emonPi used 16 x 2 I2C LCD display
```
Comment (jlark) [inline emonFirmware_test_for_relay_driver/emonPi_Serial_LCD.ino]: do we need this file in here?

File: emonFirmware_test_for_relay_driver/emonPi_Startup.ino [hunk 4]
```diff
@@ -0,0 +1,64 @@
+ 
+double calc_rms(int pin, int samples)                      //Used in emonPi startup to detect presence of AC wavefrom and estimate VRMS voltage
```
Comment (jlark) [inline emonFirmware_test_for_relay_driver/emonPi_Startup.ino]: As well  as this one?

**Comment by Basazababo** [comment]
Relay driver fimware

**Comment by Basazababo** [comment]
we don`t need this here, i just prefer to keep a copy of this as testing file because i was not able to test the functionality using emonhub.

Let me consider your comments

**Comment by jlark** [comment]
@Basazababo please see my comments

**Comment by jlark** [comment]
@Basazababo can you remove the non necassary files recommit and push?

**Comment by Basazababo** [comment]
yes, let me do it

**Comment by Basazababo** [comment]
Cleaning relay board repo

**Comment by jlark** [comment]
@Basazababo Great Honore thank you.
When you have time (later) can you fill in the Readme on this so that we don't forget how it works?

Thanks

**Comment by Basazababo** [comment]
Sure, I will do it

On Tue, May 31, 2016 at 9:52 AM, Alp Tilev notifications@github.com wrote:

> Merged #1
> https://github.com/GreatLakesEnergy/relay-board-firmware/pull/1.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/GreatLakesEnergy/relay-board-firmware/pull/1#event-676640808,
> or mute the thread
> https://github.com/notifications/unsubscribe/APCjm1cWsC87wG3bLCiKlMOCqYnbK_Hcks5qG-jBgaJpZM4IoQnU
> .

## 

_Honore Basazababo_

Electronics Engineer

+250 (0) 783.490.703

_www.gle.solar_

**Pull request merged**